docs/configuration.md

```
@@ -248,6 +249,41 @@ pipelines:
+  - name: "public-docs"
+    allow_include_sources: true
+```
+
+The option defaults to `false`, and it cannot be set under `defaults`,
+because exposing a corpus should be an explicit decision for each
+pipeline rather than something inherited by pipelines that were never
+reviewed for it.
+
+!!! warning "Any row in a configured table is effectively public"
+
+    The RAG server ships without client authentication, on the
```

markdownlint-cli2 (0.23.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)